packages/valory/skills/transaction_settlement_abci/skill.yaml

```diff
@@ -140,7 +140,7 @@ models:
       use_slashing: false
       use_termination: false
       validate_timeout: 1205
-      default_chain_name: ethereum
+      default_chain_id: ethereum
```

Comment:
breaking

Reply:
So why are we changing this? Chain id is usually an integer. Chain name is the ecosystem

Reply:
the ledger connection expects it as `chain_id`. I agree `chain_name` makes more sense.

Reply:
ok - well let's open an issue so we don't forget. Shouldn't block us now to move forward.

Reply:
https://github.com/valory-xyz/open-aea/issues/694